imagetest/test_suites/packagevalidation/package_test.go

```diff
@@ -151,15 +172,45 @@ func TestGuestPackages(t *testing.T) {
 		return
 	}
 
-	missingPkgs := findUniq(requiredPkgs, installedPkgs)
-	for _, pkg := range missingPkgs {
-		t.Errorf("Required package not installed: %s\n", pkg)
+	installedMap := make(map[string]bool)
+	for _, curr := range strings.Split(string(out), "\n") {
```

Comment:
I think `out` is undefined here, this should loop over installedPkgs, or modify `listPkgs` functions to return maps instead.

Reply:
using installedPackages should be enough (last minute rebase, lost track of this change). Thanks for pointing that out.

Reply:
done. Thank you.